### PR TITLE
DOCSUP-8215: Add aliases to functions

### PR DESCRIPTION
**Pull request by sevirov**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category:

- Documentation for #21519

### CODE REVIEW

File: docs/ru/sql-reference/functions/json-functions.md [hunk 1]
```diff
@@ -16,51 +16,65 @@ toc_title: JSON
...
 
-Проверить наличие поля с именем name.
+Проверяет наличие поля с именем `name`.
+
+Алиас: `simpleJSONHas`.
 
 ## visitParamExtractUInt(params, name) {#visitparamextractuintparams-name}
 
-Распарсить UInt64 из значения поля с именем name. Если поле строковое - попытаться распарсить число из начала строки. Если такого поля нет, или если оно есть, но содержит не число, то вернуть 0.
+Разбирает UInt64 из значения поля с именем `name`. Если поле строковое, пытается разобрать число из начала строки. Если такого поля нет, или если оно есть, но содержит не число, то вернуть 0.
```
Comment (olgarev) [inline docs/ru/sql-reference/functions/json-functions.md:25]:
```suggestion
Пытается выделить число из значения поля с именем `name`. Если поле строковое, пытается выделить число из начала строки. Если такого поля нет, или если оно есть, но содержит не число, то возвращает 0.
```

File: docs/ru/sql-reference/functions/json-functions.md [hunk 2]
```diff
@@ -16,51 +16,65 @@ toc_title: JSON
...
 ## visitParamExtractFloat(params, name) {#visitparamextractfloatparams-name}
 
 Аналогично для Float64.
 
+Алиас: `simpleJSONExtractFloat`.
+
 ## visitParamExtractBool(params, name) {#visitparamextractboolparams-name}
 
-Распарсить значение true/false. Результат - UInt8.
+Разбирает значение true/false. Результат — UInt8.
```
Comment (olgarev) [inline docs/ru/sql-reference/functions/json-functions.md:43]:
```suggestion
Пытается выделить значение true/false. Результат — UInt8.
```

File: docs/ru/sql-reference/functions/json-functions.md [hunk 3]
```diff
@@ -16,51 +16,65 @@ toc_title: JSON
...
+visitParamExtractString('{"abc":"\\u263a"}', 'abc') = '☺';
+visitParamExtractString('{"abc":"\\u263"}', 'abc') = '';
+visitParamExtractString('{"abc":"hello}', 'abc') = '';
 ```
 
-На данный момент, не поддерживаются записанные в формате `\uXXXX\uYYYY` кодовые точки не из basic multilingual plane (они переводятся не в UTF-8, а в CESU-8).
+На данный момент не поддерживаются записанные в формате `\uXXXX\uYYYY` кодовые точки не из basic multilingual plane (они переводятся не в UTF-8, а в CESU-8).
 
-Следующие функции используют [simdjson](https://github.com/lemire/simdjson) который разработан под более сложные требования для разбора JSON. Упомянутое выше предположение 2 по-прежнему применимо.
+Следующие функции используют [simdjson](https://github.com/lemire/simdjson), который разработан под более сложные требования для разбора JSON. Упомянутое выше предположение 2 по-прежнему применимо.
```
Comment (olgarev) [inline docs/ru/sql-reference/functions/json-functions.md:77]:
```suggestion
Следующие функции используют [simdjson](https://github.com/lemire/simdjson), который разработан под более сложные требования для разбора JSON. Упомянутое выше допущение 2 по-прежнему применимо.
```

**Comment by sevirov** [comment]
@kssenii Hi. Please, review this PR.

**Review by kssenii — APPROVED** [review]

**Pull request merged**